aspnetcore/blazor/components/templated-components.md

```diff
@@ -198,7 +201,92 @@ When using generic-typed components, the type parameter is inferred if possible.
 
 :::moniker-end
 
+The example provided in the `TemplatedNavBar` component (`TemplatedNavBar.razor`) assumes that the `Items` collection doesn't change after the initial render; or that if it does change, maintaining the state of components/elements used in `ItemTemplate` isn't necessary. For templated components where such usage can't be anticipated, see the [Preserve relationships with `@key`](#preserve-relationships-with-key) section.
+
+## Preserve relationships with `@key`
+
+Templated components are often used to render collections of items, such as tables or lists. In such general scenarios, we can't assume that the user will avoid stateful components/elements in the item template definition or that there won't be additional changes to the `Items` collection. For such templated components, it's necessary to preserve the relationships with the `@key` directive attribute.
+
+> [!NOTE]
+> For more information on the `@key` directive attribute, see <xref:blazor/components/key>.
+
+The following `TableTemplate` component (`TableTemplate.razor`) demonstrates a templated component that preserves relationships with `@key`:
```

Comment:
Need a period here ... I missed this, too! ........ It's ***Friday***. My 🧠 is a bit 🔥 for the week!

```suggestion
The following `TableTemplate` component (`TableTemplate.razor`) demonstrates a templated component that preserves relationships with `@key`.
```